lib/ApiRequestor.php

```diff
@@ -38,14 +43,16 @@ class ApiRequestor
      *
      * @param null|string $apiKey
      * @param null|string $apiBase
+     * @param null|mixed $appInfo
```

Comment:
Is it possible to provide a better type here than mixed? Perhaps an `array` like https://github.com/stripe/stripe-php/blob/7038876e/lib/BaseStripeClient.php#L58 for reference?

Reply:
This should be array. Type is described in BaseStripeClient -- do you want that duplicated here?

Reply:
I don't think we need a description here since the other params aren't described and I *think* `APIRequestor` is mostly internal, but up to your discretion!